packages/next-swc/crates/next-core/src/manifest.rs

```diff
@@ -78,13 +78,7 @@ impl DevManifestContentSourceVc {
         let mut routes = routes
             .into_iter()
             .flatten()
-            .map(|s| {
-                if !s.starts_with('/') {
-                    format!("/{}", s)
-                } else {
-                    s.to_string()
-                }
-            })
```

Comment:
This is no longer necessary as `.get_pathname()` will always return a pathname with a leading `/`.